doc/parameters/param_list.md

```diff
@@ -154,6 +154,8 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | Parameter | Default | Description |
 | ----------| --------| ------------|
 | controller_frequency | 20.0 | Frequency to run controller |
+| progress_checker_plugin | ["progress_checker"] | Check the progress of the robot |
```

Comment:
I think `["progress_checker"]` needs to be `"progress_checker"`?

Reply:
Yes. Fixed.